ETL/find_orphaned_case_workers.py

```
@@ -0,0 +1,250 @@
+        """
+        try:
+            logger.info("Fetching all referral collection IDs...")
+            referral_collection = self.db.collection("referral")
+            docs = referral_collection.stream()
+            
+            referral_ids = set()
+            for doc in docs:
+                referral_ids.add(doc.id)
+            
+            logger.info(f"Found {len(referral_ids)} documents in referral collection")
+            return referral_ids
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)